### DISCUSSION TITLE
Github's pages IP's showing up on blacklist.

### DISCUSSION THREAD

**Discussion by jayjayer35** [community/community#146653] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

As [before - again](https://github.com/orgs/community/discussions/143506), some IPs of github pages are blacklisted.

The following pages IPv4's are being shown as blacklisted on [spamhaus](https://check.spamhaus.org/)

185.199.109.153
185.199.110.153
185.199.111.153
185.199.108.153

As mentioned in the previous post; "For the GitHub support team - Please contact Spamhaus and request them to remove GitHub Pages' IP addresses from their listings. Thank you very much :)"

^^^

**Comment by mecodeatlas** [comment]
Hi @jayjayer35!

Thank you for taking the time to report this issue. It has been escalated to our Trust & Safety team and they will work on removing the sites causing our IP Addresses to be blacklisted. This may take some time based on our team's resources and the response time from Spamhaus.

You can try registering your domain again at a later date to see if the issue has been resolved.

We appreciate your patience and your contribution to making GitHub even better!

**Comment by sergiotocalini** [comment]
It is happening again, unfortunately I can see that it is a recurrent issue. @mecodeatlas, is there a way of having this resolve? it is really impacting the reputation of my website. Thanks in advance.

**Reply by sergiotocalini** [reply]
https://check.spamhaus.org/results?query=185.199.108.153
https://check.spamhaus.org/results?query=185.199.109.153
https://check.spamhaus.org/results?query=185.199.110.153
https://check.spamhaus.org/results?query=185.199.111.153

3 ips out of 4 are blacklisted since 2024-11-20 16:21:30 by this repo that doesn't exist anymore: https://authmex.github.io/hello.github.io/info.bin
